driver/src/orderbook/mod.rs

```diff
@@ -0,0 +1,82 @@
+use crate::contracts::stablex_contract::StableXContract;
+use crate::error::DriverError;
+
+use dfusion_core::models::{AccountState, Order};
+
+use web3::types::U256;
+
+type Result<T> = std::result::Result<T, DriverError>;
+
+pub trait StableXOrderBookReading {
+    /// Retunrs the index of the auction that is currently being solved
+    /// or an error in case it cannot get this information.
+    fn get_auction_index(&self) -> Result<U256>;
+
+    /// Returns the current state of the order book, inclducing account balances
+    /// and open orders or an error in case it cannot get this information.
+    ///
+    /// # Arguments
+    /// * `index` - the auction index for which returned orders should be valid
```

Comment:
ubernit: the notation is a bit inconsistent, `* name - description` here and `- name - description` in other places.